### DISCUSSION TITLE
False positive on sbt update

### DISCUSSION THREAD

**Discussion by magnusja** [aquasecurity/trivy#5641] [Bugs]
### Description

Having `sbt update` in a dockerfile results in

```
HIGH: The instruction 'RUN <package-manager> update' should always be followed by '<package-manager> install' in the same RUN statement.
```

It is an odd thing to assume that. I feel like that this is true for apt, but other than that? Maybe you should be explicit about the actual package manager which follow this approach.

### Desired Behavior

This error should not happen. Especially because the only thing I can do is to ignore the whole Dockerfile and not only that particular command. Or is there a way to do so?

### Actual Behavior

See description

### Reproduction Steps

```bash
1. Have a scala project
2. Use sbt
3. Call sbt update
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
I dont know man is this really necessary??
```

### Operating System

CI

### Version

```bash
Not sure runs somewhere in our CI
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by nikpivkin** [comment]
Hi @magnusja !

Thank you for the report. I created issue https://github.com/aquasecurity/trivy/issues/5643

**Reply by magnusja** [reply]
Thanks! :)
